### ISSUE TITLE
[Bug]: [PHP] Fehler: is_file(): open_basedir restriction in effect. ...deapps//core/l10n/de.js) is not within the allowed path(s):

### ISSUE THREAD

**Issue by camouflage81** [nextcloud/server#44566]
### ⚠️ This issue respects the following points: ⚠️

- [X] This is a **bug**, not a question or a configuration/webserver/proxy issue.
- [X] This issue is **not** already reported on [Github](https://github.com/nextcloud/server/issues?q=is%3Aopen+is%3Aissue+label%3Abug) OR [Nextcloud Community Forum](https://help.nextcloud.com/) _(I've searched it)_.
- [X] Nextcloud Server **is** up to date. See [Maintenance and Release Schedule](https://github.com/nextcloud/server/wiki/Maintenance-and-Release-Schedule) for supported versions.
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Bug description

Hello everyone,

I have a problem after the Nextcloud update from 28.0.3 to 28.0.4. Four new error messages are entered in the log file each time the page is called up. It doesn't matter whether I call up an app or switch to the settings. See Log.

### Steps to reproduce

1. Update Nestcloud Server from version 28.0.3 to 28.0.4.

### Expected behavior

No error in the log files

### Installation method

Community Web installer on a VPS or web space

### Nextcloud Server version

28

### Operating system

Debian/Ubuntu

### PHP engine version

PHP 8.2

### Web server

None

### Database engine version

MySQL

### Is this bug present after an update or on a fresh install?

Updated from a MINOR version (ex. 22.1 to 22.2)

### Are you using the Nextcloud Server Encryption module?

Encryption is Disabled

### What user-backends are you using?

- [X] Default user-backend _(database)_
- [ ] LDAP/ Active Directory
- [ ] SSO - SAML
- [ ] Other

### Configuration report

```shell
{
    "system": {
        "instanceid": "***REMOVED SENSITIVE VALUE***",
        "passwordsalt": "***REMOVED SENSITIVE VALUE***",
        "secret": "***REMOVED SENSITIVE VALUE***",
        "trusted_domains": [
            "cloud.***REMOVED SENSITIVE VALUE***.de"
        ],
        "datadirectory": "***REMOVED SENSITIVE VALUE***",
        "dbtype": "mysql",
        "version": "28.0.4.1",
        "overwrite.cli.url": "https:\/\/cloud.***REMOVED SENSITIVE VALUE***.de",
        "dbname": "***REMOVED SENSITIVE VALUE***",
        "dbhost": "***REMOVED SENSITIVE VALUE***",
        "dbport": "",
        "dbtableprefix": "oc_",
        "mysql.utf8mb4": true,
        "dbuser": "***REMOVED SENSITIVE VALUE***",
        "dbpassword": "***REMOVED SENSITIVE VALUE***",
        "installed": true,
        "mail_from_address": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpmode": "smtp",
        "mail_sendmailmode": "smtp",
        "mail_domain": "***REMOVED SENSITIVE VALUE***",
        "mail_smtphost": "***REMOVED SENSITIVE VALUE***",
        "mail_smtpport": "587",
        "mail_smtpauth": 1,
        "mail_smtpname": "***REMOVED SENSITIVE VALUE***",
        "mail_smtppassword": "***REMOVED SENSITIVE VALUE***",
        "default_phone_region": "DE",
        "trashbin_retention_obligation": "auto, 7",
        "activity_expire_days": 7,
        "maintenance": false,
        "log_rotate_size": 1048576,
        "updater.release.channel": "stable",
        "theme": "",
        "loglevel": 2,
        "maintenance_window_start": 1,
        "updater.secret": "***REMOVED SENSITIVE VALUE***"
    }
}
```

### List of activated Apps

```shell
Enabled:
  - activity: 2.20.0
  - bookmarks: 13.1.3
  - calendar: 4.6.7
  - circles: 28.0.0
  - cloud_federation_api: 1.11.0
  - comments: 1.18.0
  - contacts: 5.5.3
  - contactsinteraction: 1.9.0
  - dav: 1.29.1
  - deck: 1.12.2
  - federatedfilesharing: 1.18.0
  - federation: 1.18.0
  - files: 2.0.0
  - files_pdfviewer: 2.9.0
  - files_reminders: 1.1.0
  - files_sharing: 1.20.0
  - files_trashbin: 1.18.0
  - files_versions: 1.21.0
  - firstrunwizard: 2.17.0
  - geoblocker: 0.5.13
  - logreader: 2.13.0
  - lookup_server_connector: 1.16.0
  - nextcloud_announcements: 1.17.0
  - notifications: 2.16.0
  - oauth2: 1.16.3
  - password_policy: 1.18.0
  - privacy: 1.12.0
  - provisioning_api: 1.18.0
  - recommendations: 2.0.0
  - related_resources: 1.3.0
  - serverinfo: 1.18.0
  - settings: 1.10.1
  - sharebymail: 1.18.0
  - systemtags: 1.18.0
  - text: 3.9.1
  - theming: 2.3.0
  - twofactor_backupcodes: 1.17.0
  - updatenotification: 1.18.0
  - user_status: 1.8.1
  - viewer: 2.2.0
  - workflowengine: 2.10.0
Disabled:
  - admin_audit: 1.18.0
  - bruteforcesettings: 2.8.0
  - dashboard: 7.8.0 (installed 7.8.0)
  - encryption: 2.16.0 (installed 2.16.0)
  - files_external: 1.20.0 (installed 1.20.0)
  - photos: 2.4.0 (installed 2.4.0)
  - support: 1.11.1 (installed 1.11.0)
  - survey_client: 1.16.0 (installed 1.16.0)
  - suspicious_login: 6.0.0
  - twofactor_totp: 10.0.0-beta.2
  - user_ldap: 1.19.0
  - weather_status: 1.8.0 (installed 1.8.0)
```

### Nextcloud Signing status

```shell
No errors have been found.
```

### Nextcloud Logs

```shell
{"reqId":"UCFfS4UnkoGb5GxvJ34O","level":3,"time":"2024-03-29T09:11:15+00:00","remoteAddr":"***REMOVED SENSITIVE VALUE***","user":"***REMOVED SENSITIVE VALUE***","app":"PHP","method":"GET","url":"/index.php/settings/admin/serverinfo","message":"is_file(): open_basedir restriction in effect. File(/var/www/vhosts/***REMOVED SENSITIVE VALUE***.net/cloud.***REMOVED SENSITIVE VALUE***.deapps//core/js/merged-template-prepend.mjs) is not within the allowed path(s): (/var/www/vhosts/***REMOVED SENSITIVE VALUE***.net/cloud.***REMOVED SENSITIVE VALUE***.de/:/tmp/:/var/lib/php/sessions:/var/www/vhosts/***REMOVED SENSITIVE VALUE***.net/tmp) at /var/www/vhosts/***REMOVED SENSITIVE VALUE***.net/cloud.***REMOVED SENSITIVE VALUE***.de/lib/private/Template/ResourceLocator.php#100","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36 Edg/123.0.0.0","version":"28.0.4.1","data":{"app":"PHP"},"id":"660686be5edb3"}
{"reqId":"UCFfS4UnkoGb5GxvJ34O","level":3,"time":"2024-03-29T09:11:15+00:00","remoteAddr":"***REMOVED SENSITIVE VALUE***","user":"***REMOVED SENSITIVE VALUE***","app":"PHP","method":"GET","url":"/index.php/settings/admin/serverinfo","message":"is_file(): open_basedir restriction in effect. File(/var/www/vhosts/***REMOVED SENSITIVE VALUE***.net/cloud.***REMOVED SENSITIVE VALUE***.deapps//core/js/merged-template-prepend.js) is not within the allowed path(s): (/var/www/vhosts/***REMOVED SENSITIVE VALUE***.net/cloud.***REMOVED SENSITIVE VALUE***.de/:/tmp/:/var/lib/php/sessions:/var/www/vhosts/***REMOVED SENSITIVE VALUE***.net/tmp) at /var/www/vhosts/***REMOVED SENSITIVE VALUE***.net/cloud.***REMOVED SENSITIVE VALUE***.de/lib/private/Template/ResourceLocator.php#100","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36 Edg/123.0.0.0","version":"28.0.4.1","data":{"app":"PHP"},"id":"660686be5ed80"}
{"reqId":"UCFfS4UnkoGb5GxvJ34O","level":3,"time":"2024-03-29T09:11:15+00:00","remoteAddr":"***REMOVED SENSITIVE VALUE***","user":"***REMOVED SENSITIVE VALUE***","app":"PHP","method":"GET","url":"/index.php/settings/admin/serverinfo","message":"is_file(): open_basedir restriction in effect. File(/var/www/vhosts/***REMOVED SENSITIVE VALUE***.net/cloud.***REMOVED SENSITIVE VALUE***.deapps//core/l10n/de.mjs) is not within the allowed path(s): (/var/www/vhosts/***REMOVED SENSITIVE VALUE***.net/cloud.***REMOVED SENSITIVE VALUE***.de/:/tmp/:/var/lib/php/sessions:/var/www/vhosts/***REMOVED SENSITIVE VALUE***.net/tmp) at /var/www/vhosts/***REMOVED SENSITIVE VALUE***.net/cloud.***REMOVED SENSITIVE VALUE***.de/lib/private/Template/ResourceLocator.php#100","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36 Edg/123.0.0.0","version":"28.0.4.1","data":{"app":"PHP"},"id":"660686be5ed4f"}
{"reqId":"UCFfS4UnkoGb5GxvJ34O","level":3,"time":"2024-03-29T09:11:15+00:00","remoteAddr":"***REMOVED SENSITIVE VALUE***","user":"***REMOVED SENSITIVE VALUE***","app":"PHP","method":"GET","url":"/index.php/settings/admin/serverinfo","message":"is_file(): open_basedir restriction in effect. File(/var/www/vhosts/***REMOVED SENSITIVE VALUE***.net/cloud.***REMOVED SENSITIVE VALUE***.deapps//core/l10n/de.js) is not within the allowed path(s): (/var/www/vhosts/***REMOVED SENSITIVE VALUE***.net/cloud.***REMOVED SENSITIVE VALUE***.de/:/tmp/:/var/lib/php/sessions:/var/www/vhosts/***REMOVED SENSITIVE VALUE***.net/tmp) at /var/www/vhosts/***REMOVED SENSITIVE VALUE***.net/cloud.***REMOVED SENSITIVE VALUE***.de/lib/private/Template/ResourceLocator.php#100","userAgent":"Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36 Edg/123.0.0.0","version":"28.0.4.1","data":{"app":"PHP"},"id":"660686be5ed1a"}
```

### Additional info

Thanks to all

**Comment by Bertel3** [comment]
Same issue here. I just updated my nextcloud instance from version 28.0.3 to 28.0.4. There seems to be a / missing in the path before the "apps" folder.

From the log above: File(/var/www/vhosts/***REMOVED SENSITIVE VALUE***.net/cloud.***REMOVED SENSITIVE VALUE***.**deapps**//core/l10n/de.mjs)

**Comment by kesselb** [comment]
Thanks for your report.

Same as https://github.com/nextcloud/server/issues/44358 and will be fixed with 28.0.5.
